addon/components/x-select.js

```diff
@@ -180,6 +180,7 @@ export default Component.extend({
             this.attrs.onRemove(last);
           }
 
+          e.preventDefault();
```

Comment:
Shouldn't this be inside the `if` block (after `this.attrs.onRemove(last);`)? Otherwise it will prevent all backspaces.